scripts/loadTest/test.ts

```diff
@@ -0,0 +1,135 @@
+/* eslint-disable import/no-unresolved */
```

Comment:
I think we should update the configuration for this rule, such one-off overrides for eslint or tsc are not a good sign.